src/mono/wasm/runtime/export-types.ts

```diff
@@ -0,0 +1,7 @@
+import { DotNetExports } from "./exports";
+import { EmscriptenModuleConfig } from "./types";
+
+// this is the only public export from the dotnet.js module
+declare function createDotnetRuntime(moduleFactory: (api: DotNetExports) => EmscriptenModuleConfig): Promise<DotNetExports>;
```

Comment:
Maybe DotNetExports is the wrong name for this if 'createDotnetRuntime' is the only public export from dotnet.js? If the parameter name is 'api' maybe it's DotNetPublicAPI or something